lib/kramdown_rpf/kramdown.rb

```diff
@@ -164,6 +185,11 @@ def convert_print_only(el, _indent)
         raise NotImplementedError
       end
 
+      # Convert :quiz -> LaTEX (not implemented)
+      def convert_quiz(_el, _opts)
```

Comment:
If you don't care about the options, you can just use `*`:

```suggestion
      def convert_quiz(*)
```